apps/cowswap-frontend/src/modules/permit/hooks/usePreGeneratedPermitInfo.ts

```diff
@@ -19,12 +17,12 @@ export function usePreGeneratedPermitInfo(): {
 } {
   const { chainId } = useWalletInfo()
 
-  const url = `${PRE_GENERATED_PERMIT_URL}.${chainId}.json`
+  const url = ``
```

Comment:
Only to be able to test permit generation on the fly.

Revert before merging!!!

Reply:
Reverted